src/core/agent/services/multiSigService.ts

```diff
@@ -615,6 +615,46 @@ class MultiSigService extends AgentService {
     };
   }
 
+  async membersReadyToRotate(identifierId: string): Promise<string[]> {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifierId
+    );
+
+    const multiSig = await this.props.signifyClient
+      .identifiers()
+      .get(metadata.signifyName);
+
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(metadata?.signifyName);
+
+    const nextSequence = (Number(multiSig.state.s) + 1).toString();
+    const smids = members.signing;
+
+    const states: any[] = [];
+    await Promise.allSettled(
+      smids.map(async (signing: any) => {
+        const op = await this.props.signifyClient
+          .keyStates()
+          .query(signing.aid, nextSequence);
+        await waitAndGetDoneOp(this.props.signifyClient, op);
+
+        if (op.done) {
```

Comment:
If this `op` isn't done, we need to throw an error that it's not completing. Otherwise we can't continue. Please add a test for this too